connexion/apps/abstract.py

```diff
@@ -63,8 +63,9 @@ def __init__(self, import_name, api_cls, port=None, specification_dir='',
 
         logger.debug('Specification directory: %s', self.specification_dir)
 
-        logger.debug('Setting error handlers')
-        self.set_errors_handlers()
+        if not skip_error_handlers:
+            logger.debug('Setting error handlers')
+            self.set_errors_handlers()
```

Comment:
For aiohttp, the `set_error_handlers` does not actually register the exception handler. Instead the exception handling middleware is registered in `api`. So, the `skip_error_handlers` will have no effect for aiohttp. It would be nice to unify the API for Flask and Aiohttp to add the error handlers in the same way, but I don't think that fits in this PR.

Reply:
Maybe this should be a separate PR as well - Use this PR to convert all uses of `problem()` to `raise ProblemException()` and then another PR to add `skip_error_handlers`.
And then at some point someone can harmonize how the Flask and Aiohttp clients setup (or skip setting up) error handling.

Reply:
Resolving since everything appears to be passing, but let me know if you feel something else should be changed.